### DISCUSSION TITLE
Set_variable Task issue in 3.11 version onwards

### DISCUSSION THREAD

**Discussion by shekarpcs** [Netflix/conductor#3349] [General]
Set variable task is not setting variable but task is shown completed. 

We are migrating from 3.4.1 to 3.11.3.  Same workflows are working fine in 3.4.1 and set -variable is working fine. 

our process is 
Do_whilte 
http task
Set variable with httptask response
done

First loop is not setting variable. 
Second loop on words it is setting. 

are there any new settings we have to make. We have huge workflows with this logic and we are stuck with old version. 

3.10 is working fine. We want to move to latest version so that we wont fall back on versions.

**Comment by v1r3n** [comment]
Hi @shekarpcs I believe this issue has been fixed in the subsequent releases:  Can you give it a try and update if it is working as expected? 
https://github.com/Netflix/conductor/pull/3196

**Reply by shekarpcs** [reply]
Thanks for the reply. Yes that is the fix. 

But it is not merged to community version. We are stuck in production deployment. What is the plan to merge this to community version?

We are stuck at 10.7 and as 3.11 have the issues

**Reply by v1r3n** [reply]
@shekarpcs let us know if you are able to consume the latest community build.  The latest build was updated a while back.

**Comment by shekarpcs** [comment]
Ya. But latest community build is having to many critical vulnerabilities.
We cannot use it in production.

Using Netflix conductor regular version with property launcher to launch
postgres.
Purna.
